samples/enable_disable_hardware_sensors.py

```diff
@@ -0,0 +1,74 @@
+import orionsdk
+
+server = 'orion_server'
+username = 'username'
+password = 'password'
+
+# The structure of the payload required for the verbs (Enable|Disable)Sensors is not straight-forward.
+# Below is what the payload should look like:
+payload_template = [
+    {'HardwareInfoID': int, 'HardwareCategoryStatusID': int, 'UniqueName': str},
+]
```

Comment:
These 3 lines should be prefixed with `#` to make them part of the comment.  We don't want to actually create `payload_template`.

Reply:
Good Suggestion. Commented out these lines.